index.html

```diff
@@ -1012,6 +1012,158 @@ <h3>
+          "manifest">`file_handlers`</dfn></code> member is a [=list=] that
+          represents URLs within the app that handles launches of given [=file
+          types=].
+        </p>
+        <p>
+          A <dfn>file type</dfn> refers to file that has a [=mime type=] and/or
+          ends with a [=file extension=]. A <dfn>file extension</dfn> is a
+          string that start with a "." and only contains [=valid suffix code
+          points=]. A <dfn>valid suffix code point</dfn> is a [=code point=]
+          that is [=ASCII alphanumeric=], U+002B (+), or U+002E (.).
+          Additionally, [=file extensions=] are are limited to a length of 16
+          code points.
+        </p>
+        <aside class="note">
+          <p>
+            Note: These code points were chosen to support most pre-existing
+            file formats. The vast majority of file extensions are purely
+            alphanumeric, but compound extensions (such as .tar.gz) and
+            extensions such as .c++ for C++ source code are also fairly common,
+            hence the inclusion of + and . as allowed code points.
+          </p>
+        </aside>
+        <p>
+          How file handlers are registered and disambiguated is at the
+          discretion of the operating system.
+        </p>
+        <p>
+          To <dfn>process the `file_handlers` member</dfn>, given [=ordered
+          map=] |json:ordered map|, and [=ordered map=] |manifest:ordered map|:
+        </p>
+        <ol class="algorithm">
+          <li>Let |processedFileHandlers:list| be a new [=list=].
+          </li>
+          <li>Set |manifest|["file_handlers"] to |processedFileHandlers|.
+          </li>
+          <li>If |json|["file_handlers"] doesn't [=map/exist=] or
+          |json|["file_handlers"] is not a [=list=], return.
+          </li>
+          <li>[=list/For each=] |entry:ordered map| of |json|["file_handlers"]:
+            <ol>
+              <li>Let |file_handler:ordered map| be [=process a file handler
+              item=] with |entry|.
+              </li>
+              <li>If |file_handler| is failure, continue.
+              </li>
+              <li>[=list/Append=] |file_handler| to |processedFileHandlers|.
+              </li>
+            </ol>
+          </li>
+        </ol>
+        <p>
+          On [=installation=], a user agent SHOULD register the file handlers
+          with the operating system via interactions that are consistent with:
+        </p>
+        <ul>
+          <li>The [=file handler/icon=] and [=file handler/name=] appear in the
+          host operating system surfaces presenting options for opening files,
+          for example within "Open with..." context menus for associated file
+          types.
+          </li>
+          <li>The [=file handler/icon=] appears as the icon of associated files
+          in the host operating system.
+          </li>
+          <li>File types with multiple possible handlers can be disambiguated
```

Comment:
I think instead of "disambiguated" I would choose a different word here.

The OS allows the user to *select* which app should open a file, and also allows the user to set a default app.